src/schemas/openapi/OpenAPIDto.yaml

```diff
@@ -0,0 +1,14 @@
+type: object
+properties:
+  source:
+    oneOf:
+      - type: string
+      - type: object
+      - type: array
+        items:
+          type: string
+          format: binary
+  load:
+    type: boolean
```

Comment:
do you want to include this in this endpoint?
Do you want to have an endpoint that does both the conversion and loading?

Reply:
Yes, I thought it will be nice to allow the consumer of the API to load the components and the connection in the iesi repository